manager/controllers/app/fybrikapplication_controller_unit_test.go

```diff
@@ -1389,6 +1389,85 @@ func TestReadAndTransform(t *testing.T) {
 	g.Expect(plotter.Spec.Flows[0].SubFlows[0].Steps[0]).To(gomega.HaveLen(2))
 }
 
+func TestWriteNotRegisteredAsset(t *testing.T) {
```

Comment:
Looks like copy-paste from another test. You want to test here that 1. a plotter is generated successfully and 2. the asset connection in the plotter is s3 and has non-empty format.

Reply:
Fixed. Thanks